### PR TITLE
Feature/ccache gha

### PR DESCRIPTION
**Pull request by petersteneteg**
Fixes #...

Changes proposed in this PR:
 * ...
 * ...
 * ...

This has been tested on: ...

> Thank you for contributing to the Inviwo repository! Please ensure the following: 
> - Code is error and warning free
> - Code follows the Inviwo coding guidelines: 
>   https://github.com/inviwo/inviwo/wiki/Coding-Conventions
> - New Code / Processors follows the Inviwo documentation guidelines: 
>   https://github.com/inviwo/inviwo/wiki/Documentation-guide
> - New Code / processors are tested in a unittest and / or regression test: 
>   https://github.com/inviwo/inviwo/wiki/Manual#testing
> 
> Please remember to specify the environment this code been compiled and tested on:
> Operating System / Compiler / IDE / QT / CMake / Python versions.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -57,8 +57,28 @@ set(CMAKE_DISABLE_SOURCE_CHANGES ON)
 set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
 
 # verify that git submodules have been cloned
-include(cmake/verifysubmodules.cmake)
-verify_submodules("${CMAKE_CURRENT_LIST_DIR}/.gitmodules")
+if(NOT DEFINED VCPKG_INSTALLED_DIR)
+    include(cmake/verifysubmodules.cmake)
+    verify_submodules("${CMAKE_CURRENT_LIST_DIR}/.gitmodules")
+endif()
```
Comment (martinfalk) [inline CMakeLists.txt:63]: nice!

Comment (martinfalk) [reply CMakeLists.txt:63]: But doesn't that potentially backfire when using a combination of vcpkg and submodules?

File: apps/inviwodome/CMakeLists.txt
Comment (martinfalk) [inline apps/inviwodome/CMakeLists.txt:1]: Was this added on purpose?

File: apps/inviwodome/DomePreset.json
Comment (martinfalk) [inline apps/inviwodome/DomePreset.json:1]: Was this added on purpose?

File: apps/inviwodome/include/inviwo/dome/dumpfiles.h
Comment (martinfalk) [inline apps/inviwodome/include/inviwo/dome/dumpfiles.h:1]: Was this added on purpose?

File: apps/inviwodome/include/inviwo/dome/sgctmanager.h
Comment (martinfalk) [inline apps/inviwodome/include/inviwo/dome/sgctmanager.h:1]: Was this added on purpose?

File: apps/inviwodome/packaging.cmake
Comment (martinfalk) [inline apps/inviwodome/packaging.cmake:1]: Was this added on purpose?

File: CMakeLists.txt (hunk 1 above)
Comment (petersteneteg) [reply CMakeLists.txt:63]: I don't care

**Comment by petersteneteg** [comment]
note this branch is only for testing

**Pull request closed without merging**